python/cuml/test/experimental/test_explainer_permutation_shap.py

```diff
@@ -77,7 +77,7 @@ def test_regression_datasets(exact_tests_dataset, model):
 
     skmod = cuml_skl_class_dict[model]().fit(X_train, y_train)
 
-    explainer = cuml.experimental.explainer.KernelExplainer(
+    explainer = cuml.experimental.explainer.PermutationExplainer(
```

Comment:
For future tests, it would be interesting to create a synthetic model with known effects we can test - could just be OLS with predefined coefficients. Then we can exactly compute the feature contributions without having to compare against shap.